signin_parser_spec.rb

```diff
@@ -8,24 +8,35 @@
     @login = 'teacher_128'
     @password = 'teacher_128_pass'
     @enter = "Вхід"
-    @radio = 'depart_subj_spring'
     @quit = "Вийти"
   end
 
-  it "should be signed in, parsed and signed_out" do
+  it "should be signed in and sign out" do
     signed = @signin_parse.sign_in(@url, @login, @password, @enter)
-    expect(signed.class).to eq Capybara::Node::Element
-    parsed_items = @signin_parse.parse("TeacherSubject", @radio)
-    expect(parsed_items.size).to eq 69
-    parsed_items.each do |item|
-      expect(item).to match /^Дисципліна_.+/
-    end
+    expect(signed.class).to eq Capybara::Session
+    expect(signed.find('a', text: @quit).class).to eq Capybara::Node::Element
     @signin_parse.sign_out(@quit)
+    expect(signed.find_button(@enter).class).to eq Capybara::Node::Element
```

Comment:
I believe all you want to check here is that your login method returns no error with right credentials.

Reply:
No. I want to check the sign out.